forge-gui-android/pom.xml

```diff
@@ -204,18 +204,17 @@
                                 <version>6.2.2</version>
                             </dependency>
                         </dependencies>
-                        <version>4.6.1</version>
+                        <version>4.6.0</version>
```

Comment:
This is wrong. We need to use 4.6.1 

See this section in Android snapshot. 

`      - name: Install Android maven plugin
        run: |
          mkdir  -p ~/.m2/repository/com/simpligility/maven/plugins/android-maven-plugin/4.6.1
          cd ~/.m2/repository/com/simpligility/maven/plugins/android-maven-plugin/4.6.1
          curl -L -o android-maven-plugin-4.6.1.jar https://github.com/Card-Forge/android-maven-plugin/releases/download/4.6.1/android-maven-plugin-4.6.1.jar
          curl -L -o android-maven-plugin-4.6.1.pom https://github.com/Card-Forge/android-maven-plugin/releases/download/4.6.1/android-maven-plugin-4.6.1.pom`

Reply:
I believe this library change is why we don't need to mess with d8 for the snapshot build.

Reply:
Ahh that explains why i couldn't find the Version